docs/resources/upgrade_lavinmq.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: "cloudamqp"
+page_title: "CloudAMQP: cloudamqp_upgrade_lavinmq"
+description: |-
+  Invoke upgrade to latest possible upgradable versions for LavinMQ.
+---
+
+# cloudamqp_upgrade_lavinmq
+
+This resource allows you to upgrade LavinMQ version. 
+
+There is two different ways to trigger the version upgrade
+
+> - Specify LavinMQ version to upgrade to
+> - Upgrade to latest LavinMQ version
```

Comment:
@viktorerlingsson Do we want to support upgrade to latest for LavinMQ? Been testing around a bit and will require some more changes to support that. If not we can remove some part of the documentation describing this.

Reply:
No, you're right. I removed all text about upgrading to latest and re-phrased some stuff